ui/views.py

```diff
@@ -154,6 +157,32 @@ def get_context_data(self, video, **kwargs):  # pylint: disable=arguments-differ
         return context
 
 
+class VideoDownload(VideoDetail):
```

Comment:
I think we are subclassing `VideoDetail` elsewhere to reuse the `get_context_data` method and template. But here we are producing raw data so I'm not sure it's necessary to subclass that instead of a regular `View`

Reply:
✅